libsplinter/src/biome/credentials/mod.rs

```diff
@@ -21,11 +21,11 @@ pub mod database;
 mod error;
 pub(in crate::biome) mod rest_resources;
 
-use bcrypt::{hash, DEFAULT_COST};
```

Comment:
The default cost for bycrpt is pretty high and can cause login to appear to hang for a while, you might want to consider lowering it or making it configurable.

Reply:
I added a commit to make the hash cost configurable